flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/subtask/job-checkpoints-subtask.component.html

```diff
@@ -91,6 +91,7 @@
         <th nzSortKey="alignment.processed" nzShowSort><strong>Processed (persisted) Data</strong></th>
         <th nzSortKey="alignment.duration" nzShowSort><strong>Alignment Duration</strong></th>
         <th nzSortKey="start_delay" nzShowSort><strong>Start Delay</strong></th>
+        <th nzSortKey="unaligned_checkpoint" nzShowSort><strong>Unaligned Checkpoint</strong></th>
```

Comment:
nit: invert to Aligned to save space?

Reply:
you sure...? The meaning of "Aligned" is ambiguous, and I do not think it is straightforward to users that "Aligned" means "not completed as an unaligned checkpoint"?

Reply:
We could add a tooltip (and it should be documented).
I'm also fine with "Unaligned Checkpoint".